crates/aptos/src/genesis/mod.rs

```diff
@@ -131,13 +131,14 @@ pub fn fetch_genesis_info(git_options: GitOptions) -> CliTypedResult<GenesisInfo
         ));
     }
 
-    let modules = client.get_modules("framework")?;
+    // Should the target be configurable?
+    let framework = client.get_framework()?;
```

Comment:
We made it not configurable, considering it only happens once.

Reply:
Done